### PR TITLE
Added requirements.txt.

### PR DESCRIPTION
**Pull request by Shalmezad**
Will help later with a docker setup.

### CODE REVIEW

**Comment by philipcmonk** [comment]
Based on discussion here, should we maybe specify exact versions?
 https://github.com/numerai/submission-criteria/issues/1#issuecomment-329207003

**Comment by Shalmezad** [comment]
Can do. Still playing with the repo so not sure the versions I have will work yet. If you post a `pip list`, I'll go ahead and lock down the versions for the packages I have listed.

**Comment by lorenzofar** [comment]
In the committed file there are some missing modules,these should be all the required modules along with their version numbers:

requests==2.12.4
pandas==0.20.3
setuptools==33.1.1
numpy==1.11.0
bottle==0.12.13
pymongo==3.5.1
pqueue==0.1.7
boto3==1.4.7
botocore==1.7.9
scipy==0.19.1
scikit_learn==0.19.0

**Comment by Shalmezad** [comment]
Thanks. Adding in now. 
Would you mind double checking pandas though? Noticed that setup.py specifies 0.20.2.

**Comment by lorenzofar** [comment]
From pandas website (referring to 0.20.3):

> This is a minor bug-fix release in the 0.20.x series and includes some small regression fixes and bug fixes. We recommend that all users upgrade to this version.

I don't know however whether there are specific reasons not to upgrade

**Comment by philipcmonk** [comment]
Upgrading should be fine, but the setup.py file should also be updated.

File: requirements.txt [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
 # required by s3_util.py
-boto3
+boto3 == 1.4.7
 
 # required by database_manager.py
-pymongo
-pandas
-sklearn
+pymongo == 3.5.1
+pandas == 0.20.2
```
Comment (philipcmonk) [inline requirements.txt]: No trailing whitespace, please.

**Comment by philipcmonk** [comment]
Awesome, thanks!

**Comment by philipcmonk** [comment]
And definitely looking forward to a Docker setup.

**Pull request merged**